### PR TITLE
include checkpoint parsing for PAM failed logins

### PR DESCRIPTION
**Pull request by verbecee**
## Description
I added in parsing to map fields to ECS, specifically for checkpoint incorrect password attempts.

## Todos
- [ ] 
- [ ]

### CODE REVIEW

File: config/processors/syslog_log_audit_checkpoint.operations.conf [hunk 1]
```diff
@@ -30,22 +30,30 @@ filter {
...
-    
+
     grok {
       tag_on_failure => "_parsefailure_auth"
         match => { "rest_msg" => "^(?<event.created>.*?\d+:\d+:\d+) (?<observer.name>.*?) .*?\): (?<rule.description>.*?);.*?uid=(?<user.id>\d+).*?tty=(?<network.protocol>.*?) .*? user=(?<user.name>.*?)$" }
     }
   }
+
+  if [rest_msg] =~ "illegal pam password attempt" {
+    dissect{
```
Comment (KrishnanandSingh) [inline config/processors/syslog_log_audit_checkpoint.operations.conf:43]: Should be
```suggestion
    dissect {
```

File: config/processors/syslog_log_audit_checkpoint.operations.conf [hunk 2]
```diff
@@ -30,22 +30,30 @@ filter {
...
+		mapping => {
```
Comment (KrishnanandSingh) [inline config/processors/syslog_log_audit_checkpoint.operations.conf:44]: Please follow 2 space indentations.

**Review by KrishnanandSingh — CHANGES_REQUESTED** [review]

File: config/processors/syslog_log_audit_checkpoint.operations.conf (hunk 2 above)
Comment (verbecee) [reply config/processors/syslog_log_audit_checkpoint.operations.conf]: Done!

**Review by KrishnanandSingh — APPROVED** [review]
LGTM

**Pull request merged**